### ISSUE TITLE
Recognition 

### ISSUE THREAD

**Issue by HarshalPal** [PaddlePaddle/PaddleOCR#9669]
I've been tasked to train a model for recognizing hand-written text along with usual printed texts in english. The pretrained weights (PP OCR v3) as such can recognize the printed ones pretty well, so i trained them on handwritten texts after obtaining data from kaggle.

After training on 4000 train images and 4000 validation imgs for 10 epochs, i hadn't expected much on handwritten text, but surprisingly,  the inference model (obrained after training and exporting) is doing worse on printed texts. Infact recognition of printed texts were better in pretrained model. During training, i get to see 0.85+ accuracy.

Can somebody point where am i wrong?

**Comment by TuanBC** [comment]
At first, you should evaluate all 10 output models to see how it performs on printed text. It could bring more insight to why this problem happens. In my opinion, if you want to build a general OCR model for both handwritten and printed text, you should definately add some more printed text in the training dataset. Or if you want to try, then try freezing 1 pretrain model when you are training in distillation configuration.

**Comment by HarshalPal** [comment]
Let me elaborate a bit about my approach. My assumption is that since the pre-trained weights can recognize printed texts pretty well, training on just handwritten texts would result into an OCR capable of recognizing both printed and handwritten ones. So my training data only contained handwritten texts. 

Was this assumption wrong?

**Comment by an1018** [comment]
Training with new samples, results in poor effect on the printed data; You can reduce train epoch; And how does the newly trained model work with handwriting texts?

**Comment by TuanBC** [comment]
@HarshalPal It will "forget" the original training set containing the printed text if you train only handwritting text for too long. That's normal and that's why you should add more printed text to the training set, whether they are synthesized or real data.
